lib/commands/draft.js

```diff
@@ -73,9 +71,18 @@ module.exports = async function (args) {
   logger.debug('New semver to apply: %s', release)
 
   const newVersion = releaseBuilder.bump(release)
-  logger.info('Applied %s version to project: %s', release, newVersion)
+  logger.info('Calculating %s version to project it should be %s', release, newVersion)
+
+  const draft = releaseBuilder.build()
+  console.log(draft.message)
+
+  // TODO
+  const fileTmp = await tempWrite(draft.message, 'edit-release-message.md')
+  console.log({ fileTmp })
+
+  require('child_process').spawnSync('start', [fileTmp])
```

Comment:
require at the top

Reply:
Ops, it is a misprint: the edit of release is only in the publish command